package.json

```diff
@@ -18,6 +18,7 @@
   "license": "MIT",
   "devDependencies": {
     "coffee-script": "^1.9.1",
+    "docha": "git://github.com/thompsnm/docha.git",
```

Comment:
I saw you had a commit-ish on this at one point. Its a good practice to keep that rather than pointing to the head of master. Also, since this is on Github you can use the shortcut url if you like.

`"docha": "thompsnm/docha#36f8d21"`

https://docs.npmjs.com/files/package.json#git-urls-as-dependencies